docs/specification.md

```diff
@@ -841,11 +841,13 @@ The DAO contract itself won't store the metadata, rather a dedicated contract wi
 Motivation:
 * The baseDAO contract is very large and approaches the Tezos hard limits.
   * Providing the metadata during origination would significantly reduce the amount of functionality we can put into contract;
-  * Inserting the metadata after origination still requires mechanisms to manage the metadata;
-    this way there is no guarantee that it remains constant (as users will probably expect).
+  * Inserting the metadata after origination still requires mechanisms to manage the metadata, increasing the contract size.
 
 ## Deployment with metadata
 
+The deployment of contract with metadata has an extra step: a dedicated contract for carrying metadata has to be originated first.
+Then the baseDAO contract should include the reference to a metadata key in the contract in order to be compliant with TZIP-16.
+
 The proposed way of contract deployment is the following:
 
 ### 1. Construct baseDAO metadata
```

Comment:
So now it's in two places, is it intended?

Reply:
Wtf :fuu: